source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,60 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Group elastic profile by plugin-id ([#3772](https://github.com/gocd/gocd/issues/3772))'%>
+* <%= link_to_commit 'ca7c3fa232087b67553e7d30bbfb409ba5aab49d', 'Add a request processor to allow plugins to query the server info ([#3401](https://github.com/gocd/gocd/issues/3401))'%>
+* <%= link_to_commit '1c1f2fe0d4ebda46ddfe0766ce50d580039a3f77', 'Admins can edit pipelines from VSM now'%>
+* <%= link_to_commit 'd47180f2193239ddd318157c43d3f60b6168d9e0', 'Add a few more fields to materials form which is used in creation of pipelines.'%>
+
+<h4>API Changes</h4>
+* <%= link_to_commit '0587f21042701d9ee1e64ae698341c495df6c8d8', 'Plugin info api ([#3719](https://github.com/gocd/gocd/issues/3719))'%>
+* <%= link_to_commit '19b3e9c800a1a8b3e7b6d2049fe3382dac671d12', 'Add plugin settings metadata to plugin info V3'%>
```

Comment:
I believe that these two are part of the same "story", no need to have individual links?